src/chains/ethereum/ethereum/src/data-managers/block-manager.ts

```diff
@@ -272,26 +272,44 @@ export default class BlockManager extends Manager<Block> {
     await this.#blockIndexes.put(LATEST_INDEX_KEY, number);
   }
 
+  async getEarliest() {
+    const fallback = this.#blockchain.fallback;
+    if (fallback) {
+      const json = await fallback.request<any>("eth_getBlockByNumber", [
+        Tag.earliest,
+        true
+      ], { disableCache: true });
```

Comment:
I don't think we should be disabling the cache here. Maybe I could be convinced, though! What was your reasoning here?